### DISCUSSION TITLE
Issue running FlipperNested in terminal

### DISCUSSION THREAD

**Discussion by TomShook1** [AloneLiberty/FlipperNested#15] [Q&A]
Hello,
I am having trouble running "FlipperNested" in Terminal.
I type: `python -m FlipperNested` 
and nothing happens.
No error message saying that the file does not exist, nothing.
Wondering if someone could help.

Update:
when the Flipper is not plugged in and I try to execute, the terminal spits out "Flipper missing" message.
when qFlipper is open (and flipper is connected) and I try to execute, it gives me a permission error.
Please help
Thanks.
(I'm VERY new to this)

**Comment by AloneLiberty** [comment]
If you still have this issue, please create issue in https://github.com/AloneLiberty/FlipperNestedRecovery with OS and Python info.
